### PR TITLE
Correct grammar in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by shockone**
@G07cha, please review. Answer "Approved." or something, if you agree with the changes.

### CODE REVIEW

**Comment by qaisjp** [comment]
> Awesome, but hold on! First of all, [make sure](https://github.com/black-screen/black-screen/issues) someone else hasn't previously found this bug, and, if no related issues are found, let's gather as much information as possible.

> if no related issues are found, let's gather as much information as possible.

> **let's** gather as much information as possible

lets

**Comment by shockone** [comment]
@qaisjp, are you sure? I've just googled it, and they say it's a contraction from "let us", and should be written "let's". Could you please provide a proof?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,30 +1,27 @@
 I've found a bug!
```
Comment (G07cha) [inline CONTRIBUTING.md]: `I have found a bug` becouse second section named `I have some important changes`

**Comment by qaisjp** [comment]
Ah, that's embarrassing, I misread the context.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,30 +1,27 @@
...
------
-Awesome, but hold on! First of all, make sure that someone else hasn't found this bug previously, previously, and, if no related issues are found, let's gather as much information as possible.
+-----------------
 
-1. Make sure that you are using the latest version.
+Awesome, but hold on! First of all, [make sure](https://github.com/black-screen/black-screen/issues) someone else hasn't previously found this bug, and, if no related issues are found, let's gather as much information as possible.
 
+1. Make sure you are using the latest version of the application:
```
Comment (G07cha) [inline CONTRIBUTING.md]: Add new line becouse code-block messed up

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,30 +1,27 @@
...
- 4.1. Open View -> Toggle Developer Tools
- 
- 4.2. Find Console
- 
- 4.3. Copy output and paste to the issue
- 
+2. If the bug is still present, [open an issue](https://github.com/black-screen/black-screen/issues/new).
+3. Write steps to reproduce the bug.
+4. Take some screenshots.
+5. Gather debug logs.
```
Comment (G07cha) [inline CONTRIBUTING.md]: Add newline here please

**Comment by G07cha** [comment]
Actually I think that for grammar check we should find native english speaker.

**Comment by qaisjp** [comment]
Native english speaker here

**Comment by G07cha** [comment]
@shockone, sublist still messed up becouse no newlines added after each point

**Comment by shockone** [comment]
@G07cha, fixed.

**Pull request merged**